### PR TITLE
[SPARK-17642] [SQL] [FOLLOWUP] drop test tables and improve comments

### PR DESCRIPTION
**Pull request by wzhfy**
## What changes were proposed in this pull request?

Drop test tables and improve comments.

## How was this patch tested?

Modified existing test.

### CODE REVIEW

**Comment by wzhfy** [comment]
cc @cloud-fan

**Comment by cloud-fan** [comment]
LGTM, please also address one more comment that drop testing tables.

File: sql/core/src/test/resources/sql-tests/inputs/describe-table-column.sql [hunk 1]
```diff
@@ -13,6 +13,8 @@ DESC FORMATTED desc_col_temp_table desc_col_temp_table.key;
 -- Describe a non-existent column
 DESC desc_col_temp_table key1;
 
+DROP VIEW desc_col_temp_table;
```
Comment (cloud-fan) [inline sql/core/src/test/resources/sql-tests/inputs/describe-table-column.sql]: since it's a view, can we name it `desc_col_temp_view`?

Comment (cloud-fan) [reply sql/core/src/test/resources/sql-tests/inputs/describe-table-column.sql]: and it's better to put all `DROP` commands at the end.

Comment (wzhfy) [reply sql/core/src/test/resources/sql-tests/inputs/describe-table-column.sql]: ok. I'll fix it right now.

**Comment by cloud-fan** [comment]
thanks, merging to master!

**Pull request closed without merging**